src/app/functions/universal/dataBrokerNames.ts

```diff
@@ -0,0 +1,199 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export const dataBrokerNamesRecord: Record<string, string> = {
+  "411locate.com": "411 Locate",
+  "peoplefinders.com": "People Finders",
+  "peoplesearchnow.com": "People Search Now",
+  "privateeye.com": "Private Eye",
+  "usa-people-search.com": "Usa People Search",
+  "advancedbackgroundchecks.com": "Advanced Background Checks",
+  "publicrecordsnow.com": "Public Records Now",
+  "advanced-people-search.com": "Advanced People Search",
+  "usphonebook.com": "US Phone Book",
+  "golookup.com": "Go Lookup",
+  "findpeoplesearch.com": "Find People Search",
+  "fastpeoplesearch.com": "Fast People Search",
+  "smartbackgroundchecks.com": "Smart Background Checks",
+  "searchpeoplefree.com": "Search People Free",
+  "cyberbackgroundchecks.com": "Cyber Background Checks",
+  "usatrace.com": "Usa Trace",
+  "quickpeopletrace.com": "Quickpeople Trace",
+  "fastbackgroundcheck.com": "Fast Background Check",
+  "cellrevealer.com": "Cell Revealer",
+  "propeoplesearch.com": "Pro People Search",
+  "callercenter.com": "Caller Center",
+  "familytreenow.com": "Family Tree Now",
+  "truepeoplesearch.com": "Truepeople Search",
+  "instantcheckmate.com": "Instant Checkmate",
+  "truthfinder.com": "Truth Finder",
+  "ussearch.com": "Us Search",
+  "zabasearch.com": "Zaba Search",
+  "intelius.com": "Intelius",
+  "addresses.com": "Addresses",
+  "yellowpages.com": "Yellow Pages ",
+  "onlinesearches.com": "Online Searches",
+  "anywho.com": "AnyWho",
+  "people.yellowpages.com": "The Real White Pages",
+  "reversephonelookup.com": "Reverse Phone Lookup",
+  "peoplesmart.com": "People Smart",
+  "yellowbook.com": "YellowBook",
+  "beenverified.com": "Been Verified",
+  "open-public-records.com": "Open Public Records",
+  "peoplelooker.com": "People Looker",
+  "knowthycustomer.com": "Know Thy Customer",
+  "searchsystems.net": "Search Systems",
+  "neighborwho.com": "Neighbor Who",
+  "numberguru.com": "Number Guru",
+  "ownerly.com": "Ownerly",
+  "bumper.com": "Bumper",
+  "radaris.com": "Radaris",
+  "pub360.com": "Pub360",
+  "backgroundcheck.run": "Background Check",
+  "newenglandfacts.com": "New England Facts",
+  "veripages.com": "VeriPages",
+  "people-background-check.com": "People Background Check",
+  "centeda.com": "Centeda",
+  "dataveria.com": "DataVeria",
+  "verecor.com": "Verecor",
+  "veriforia.com": "Veriforia",
+  "councilon.com": "Councilon",
+  "findrec.com": "FindRec",
```

Comment:
Spot checked this one and got:
```
410 Gone
nginx/1.25.0
```

Reply:
Leaving as is, as it was listed from OneRep for now.